arches/app/models/models.py

```diff
@@ -1570,6 +1570,19 @@ class Meta:
         db_table = "plugins"
 
 
+class WorkflowHistory(models.Model):
+    workflowid = models.UUIDField(primary_key=True)
+    stepdata = JSONField(null=False, default=dict)
+    componentdata = JSONField(null=False, default=dict)
```

Comment:
I decided to split the workflow data into two JSON fields since I saw that the requests were being made separately by separate frontend viewmodels. So this way those viewmodels can patch in partial data without needing to know or care about the rest of the data.